example.fi

```diff
@@ -104,3 +104,10 @@ fn interval(start: Int, finish: Int) : {:error, String} | {:ok, Int} do
     {:error, "Interval cannot end before it's started"}
   end
 end
+
+# Wrap and use an external function defined in the BEAM
+ext str_length(str: String) : Int = {"Elixir.String", "length", [str]}
```

Comment:
Maybe we could add examples for calling Erlang modules as well, for more understanding of the usage, although this is basically a literal MFA call

Reply:
```elixir
ext list_size(l: List[Int]) : Int = {"erlang", "length", [l]}
```